### PR TITLE
Fixed userid comparison and subscribe websockets.

### PR DESCRIPTION
**Pull request by dughetti**
We fixed userid comparison problem and updated subscribe websocket to new version of sails.

### CODE REVIEW

File: api/controllers/ServiceController.js [hunk 1]
```diff
@@ -80,8 +80,8 @@ module.exports = {
 	subscribe: function(req, res) {
 		var id = [];
 		id.push(req.param('user'));
-		Activity.subscribe(req.socket);
-		Activity.subscribe(req.socket, id);
+		Activity.watch(req);
+		Activity.subscribe(req.socket, [id]);
```
Comment (lsvx) [inline api/controllers/ServiceController.js]: Can you change `Activity.subscribe(req.socket, [id])` to just `Activity.subscribe(req, [id])`? https://github.com/balderdashy/sails-docs/blob/master/reference/websockets/resourceful-pubsub/subscribe.md

Comment (dughetti) [inline api/controllers/ServiceController.js]: Maybe.. the example on the documentation is the following 
```javascript
User.find({}).exec(function(e,listOfUsers){
        User.subscribe(req.socket,listOfUsers,['create','destroy']);
 });
````
I kept the same.

Comment (lsvx) [inline api/controllers/ServiceController.js]: Hmm they must allow both types of arguments.

**Comment by lsvx** [comment]
Merge this bad boy!

**Pull request merged**